dev/templates/item.html

```diff
@@ -1,4 +1,4 @@
-<div class="hit">
+<div class="hit" id="hit-{{objectID}}">
```

Comment:
Testing purpose, I use a second hits widget in our kitchen sink to make a summary of the results, and I wanted to make links :)